README.md

```diff
@@ -32,6 +32,8 @@ Tool that generates custom [Angular-Material v1.x](http://github.com/angular/mat
 | ----------------------- | ----------- | -------------------------------------------------------------------------- |
 | `destination` (*)       | `string`    | Target location for the Material build.                                    |
 | `modules`               | `string[]`  | Modules that should be part of the build.<br/> All modules will be built if nothing is specified.                                                                                                           |
+| `exclude-modules`               | `string[]`  | Modules that should be excluded from the build.<br/> Use to exclude e.g. core modules when building a bundle that should get lazy-loaded.                                                                                                           |
+| `exclude-main-module`               | `boolean`  | Set to `true` to exclude the code for the Angular Material main module. Use when building a bundle that will get lazy-loaded and extend an already existing main module.                                                                 |
```

Comment:
These field names should be camelCased rather than snake-cased.

Reply:
@Splaktar What about the other existing options `main-filename`, `destination-filename`? Both use snake case.

Reply:
Option names should be lower camel case. I also see `mainFilename` or `destinationFilename` in camel case below?

Reply:
@devversion true. Changed it.